### ISSUE TITLE
check out nextflow

### ISSUE THREAD

**Issue by vsoch** [singularityware/singularity-hub#11]
[http://www.nextflow.io/](http://www.nextflow.io/)

Another option for workflows - this one uses cwl instead of wdl.

**Comment by pditommaso** [comment]
Hi there, no nextflow does not use cwl nor wdl. It implements its own DSL.

**Comment by vsoch** [comment]
yeah my colleague suggested this, I wanted to minimally look into it! Thanks for this information!

**Comment by pditommaso** [comment]
It a great tool, of course. Not coincidentally I'm the author ;)
